### PR TITLE
Add Question 4: What is the address where you need support?

### PR DESCRIPTION
**Pull request by huwd**
# What
@ TODO
# Screenshots
**mobile**
image

**desktop**
image

### Validation

**mobile**
image

**desktop**
image

### CODE REVIEW

File: app/controllers/coronavirus_form/support_address_controller.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  def validate_conditionally_present_fields(product)
+    return [] if product["postcode"].present? || product["county"].present?
+
+    invalid_fields = []
+
+    if product["postcode"].blank?
+      invalid_fields << {
+        field: "postcode",
+        text: t("coronavirus_form.errors.missing_postcode_or_county_fields"),
```
Comment (issyl0) [inline app/controllers/coronavirus_form/support_address_controller.rb]: Given we have both postcode and county fields, this two-in-one error message feels confusing. 🤔

File: config/locales/en.yml [hunk 2]
```diff
@@ -123,3 +124,16 @@ en:
         label: Middle names (optional)
       last_name:
         label: Last name
+    support_address:
+      title: What is the address where you need support?
+      hint: If you're applying for someone else, enter the address of the person who needs support.
```
Comment (issyl0) [inline config/locales/en.yml]:
```suggestion
      hint: If you’re applying for someone else, enter the address of the person who needs support.
```

File: app/controllers/coronavirus_form/support_address_controller.rb (hunk 1 above)
Comment (huwd) [reply app/controllers/coronavirus_form/support_address_controller.rb]: ah i see, like keep the message but split it out?
I'm still waiting on final copy - but the deal is you must have at least one of them

Comment (huwd) [reply app/controllers/coronavirus_form/support_address_controller.rb]: i've amended that to split them out, hopefully a bit less confusing?

**Comment by huwd** [comment]
Also. thanks to issy's suggestion, noticed the hint text wasn't rendering (see the screenshots).
That's now fixed

**Review by bilbof — APPROVED** [review]

**Review by issyl0 — APPROVED** [review]
Thanks Huw!

**Pull request merged**